### ISSUE TITLE
Select & Combobox don't work

### ISSUE THREAD

**Issue by mcasto** [wotamann/vuetify-form-base#41]
First, I _love_ this component. I was just about to homebrew something of my own like it, then I thought, "I should check if anyone has already invented that wheel." I'm glad I did.

I do have a problem, though. When I set an element's type property to "select" or "combobox," I get an error saying "Unknown custom element: `<v-select>`" (or `v-combobox`).

My schema looks like this:
```
{
	initiativeBonus: {
		type: "number",
		label: "Initiative Bonus",
		col: 12
	},
	addWeapon: {
		type: "text",
		prependIcon: "mdi-plus-circle",
		ext: "hidden",
		flat: true,
		solo: true,
		col: 1,
	},
	weapons: {
		type: "array",
		col: 11,
		schema: {
			name: {
				type: "text",
				label: "Name",
			},
			type: {
				type: "select",
				label: "Type",
				items: [
					"Simple Melee",
					"Simple Ranged",
					"Martial Melee",
					"Martial Ranged",
				],
			},
			range: {
				type: "text",
				label: "Range",
			},
			damageType: {
				type: "combobox",
				label: "Damage Type",
				items: [
					"Acid",
					"Bludgeoning",
					"Cold",
					"Fire",
					"Force",
					"Lightning",
					"Necrotic",
					"Piercing",
					"Poison",
					"Psychic",
					"Radiant",
					"Slashing",
					"Thunder",
				],
			},
			bonuses: {
				type: "group",
				label: "Bonuses",
				col: 12,
				schema: {
					toHit: {
						type: "number",
						label: "To Hit",
					},
					toDamage: {
						type: "number",
						label: "To Damage",
					},
				},
			},
			damage: {
				type: "text",
				label: "Damage",
				hint: "1d8, 2d10, ...",
			},
			link: {
				type: "text",
				label: "Info Link",
			},
		},
	},
}
```

**Comment by wotamann** [comment]
I have tested this scheme and it works properly in a test. The error 'unknown custom element' may indicate that your Vuetify element 'v-select' is not correctly initialized.  

Does a simple scheme work using 'v-select'?

**Comment by issasami** [comment]
Hi! I had the same problem ( 

When i try to di something like this 
`"priority": {
        "type": "combobox",
        "items": [],
        "multiple": true,
        "label": "Priority"
      },`

I got the same error

"vuetify": "2.3.4",
"vuetify-form-base": "0.1.22",

**Comment by wotamann** [comment]
> "vuetify": "2.3.4",
> "vuetify-form-base": "0.1.22",

You could try [selection.vue](https://github.com/wotamann/vuetify-form-base/blob/master/example/src/components/selection.vue) from example. You can check your enviroment, by using this tested and working sample

**Comment by jorgv** [comment]
same problem here, following the selection example, using 
"vuetify": "2.3.4",
"vuetify-form-base": "^0.1.22",

In nuxt:
"nuxt": "^2.13.0",
"@nuxtjs/vuetify": "^1.11.2",

**Comment by jorgv** [comment]
If you are using Nuxt I think this is the solution: [#2](https://github.com/wotamann/vuetify-form-base/issues/2#issuecomment-577365364)

**Comment by issasami** [comment]
> If you are using Nuxt I think this is the solution: [#2](https://github.com/wotamann/vuetify-form-base/issues/2#issuecomment-577365364)

This is not a solution to my problem. I do not use Nuxt. 

Issue still reproduce with Vue + Vuetify

**Comment by jorgv** [comment]
@issasami But did you tried at least? 
Directly importing the components in the Vue App

```
`import Vuetify, { VLayout, VCheckbox, VSelect } from "vuetify/lib";

Vue.use(Vuetify, {
components: { VLayout, VCheckbox, VSelect },
});`

```

The nuxt solution was to include it in a plugin but it should work directly in the Vue app where you initialize the app.

**Comment by issasami** [comment]
@jorgv Yes. Works with direct import.

But as I understand it this is not how it should work

**Comment by Mashiane** [comment]
This looks interesting, someone is developing something "game"related. Wow!

**Comment by mcasto** [comment]
I still can't get it to work ... but, I did figure out something that might put someone else (maybe even a future version of me) on the right track:

First, I'm running a project built from vue-cli built with:
`vue create [project-name]`
`vue add vuetify`

Now ... setting vuetify-form-base aside, I just entered:
`<component :is="'v-select'" />`

and I got the same error: `Unknown custom element: <v-select> ...`

_Note: now I'm just thinking "out loud" through my keyboard because maybe it'll trigger an idea for me or someone else._

So, now I think this isn't a problem with vuetify-form-base but, rather, a bug in Vue itself. If it's a Vue bug, then why does the example code work? 

It looks like the example didn't install Vuetify with `vue add vuetify` ... it looks like vuetify was installed manually in the example. So ... maybe it's a bug in how `vue add ...` installs the plugin.

I just ran the following:
`vue create --default component-test`

In App.vue, I added `<component :is="'v-select'" />` and I got the same error. That leads me to believe it _is_ a bug with Vue, so I'll go mention it there, but, given this info, I'm completely baffled as to why the example works. Any thoughts, @wotamann ?
